src/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -284,6 +367,74 @@ export class MongoEntityConverter {
     return mongoPieces.map((mongoPiece) => this.convertPiece(mongoPiece))
   }
 
+  public convertToMongoPiece(piece: Piece): MongoPiece {
+    let enable: { start: number, duration?: number} = {
+      start: piece.getStart()
+    }
+    if (piece.duration) {
+      enable = {
+        ...enable,
+        duration: piece.duration
+      }
+    }
```

Comment:
We should avoid as much as possible using mutable variables. It looks like this can be rewritten as:
```typescript
     const enable: MongoPiece['enable'] = {
       start: piece.getStart()
       duration: piece.duration
     }
```

Reply:
Updated